pynamodb/tests/test_mypy.py

```diff
@@ -9,6 +9,18 @@
 from .mypy_helpers import assert_mypy_output  # noqa
 
 
+def test_model():
```

Comment:
`test_model_count`?

Reply:
I figured I'd add more model-related tests in here later as necessary